### DISCUSSION TITLE
How to use Github efficiently?

### DISCUSSION THREAD

**Discussion by gitshiven** [community/community#36177] [Other Feature Feedback, Questions, & Ideas] [answered]
Teach Github.

**Comment by FahdSeddik** [comment, marked as answer]
Hey @gitshiven 
This [link](https://missing.csail.mit.edu/2020/version-control/) will guide you through most of the things you will need. This material is provided by MIT with notes and video explanation. Hope this helps!

**Reply by FahdSeddik** [reply]
Make sure to mark this as an answer so people can see it easily
